lib/utils/is-electron.ts

```diff
@@ -0,0 +1,9 @@
+// https://github.com/atom/electron/issues/22
+
+export const isElectron: () => boolean = () => {
+  debugger;
+  return !!window?.process?.type;
+};
+
+export const isMacElectron: () => boolean = () =>
```

Comment:
the name `isMacApp` or `isMac` seems like a closer match to what this function returns and to what we want to know. what are your thoughts on renaming it?